src/surbhi/BankingOperation.java

```diff
@@ -0,0 +1,182 @@
+			kBal += amount;
+			kCreditCnt++;
+			System.out.println("Amount Rs." + amount + " Credited in KOTAK account, Now total balance " + kBal);
+			break;
+		default:
+			System.out.println("Please enter amount more then Zero, Thank You..!!");
+		}
+	}
+
+	void bankDebit(String bankName, int amount) {
+		switch (bankName) {
+		case "ICICI":
+			if (amount <= iBal) {
+				iBal -= amount;
+				iDebitCnt++;
+				System.out.println("Amount Debited from ICICI Account " + amount + ", Now Current Balance " + iBal);
+			} else
+				System.out.println("Insufficient Balance, Current Balance in ICICI Account " + iBal);
+			break;
+		case "HDFC":
+			if (amount <= hBal) {
+				hBal -= amount;
+				hDebitCnt++;
+				System.out
+						.println("Amount Debited from HDFC Account is Rs." + amount + ", Now Current Balance " + hBal);
+			} else
+				System.out.println("Insufficient Balance, Current Balance in HDFC Account " + hBal);
+			break;
+		case "SBI":
+			if (amount <= sBal) {
+				sBal -= amount;
+				sDebitCnt++;
+				System.out.println("Amount Debited from SBI Account " + amount + ", Now Current Balance " + sBal);
+			} else
+				System.out.println("Insufficient Balance, Current Balance in SBI Account " + sBal);
+			break;
+		case "KOTAK":
+			if (amount <= kBal) {
+				kBal -= amount;
+				kDebitCnt++;
+				System.out.println("Amount Debited from KOTAK Account " + amount + ", Now Current Balance " + kBal);
+			} else
+				System.out.println("Insufficient Balance, Current Balance in KOTAK Account " + kBal);
+			break;
+		default:
+			System.out.println("Operation Aborted..Enter Valid details, Please..!!");
+		}
+	}
+
+	void individualBankDebitOperations(String bankName) {
+		switch (bankName) {
+		case "ICICI":
+			System.out.println("Total No. of Debit Operations in ICICI Bank account " + iDebitCnt);
+			break;
+		case "HDFC":
+			System.out.println("Total No. of Debit Operations in HDFC Bank account " + hDebitCnt);
+			break;
+		case "SBI":
+			System.out.println("Total No. of Debit Operations in SBI Bank account " + sDebitCnt);
+			break;
+		case "KOTAK":
+			System.out.println("Total No. of Debit Operations in KOTAK Bank account " + kDebitCnt);
+			break;
+		default:
+			System.out.println("Please enter valid details.");
+		}
+	}
+
+	void individualBankCreditOperations(String bankName) {
+		switch (bankName) {
+		case "ICICI":
+			System.out.println("Total No. of Credit Operations in ICICI Bank account " + iCreditCnt);
+			break;
+		case "HDFC":
+			System.out.println("Total No. of Credit Operations in HDFC Bank account  " + hCreditCnt);
+			break;
+		case "SBI":
+			System.out.println("Total No. of Credit Operations in SBI Bank account " + sCreditCnt);
+			break;
+		case "KOTAK":
+			System.out.println("Total No. of Credit Operations in KOTAK Bank account " + kCreditCnt);
+			break;
+		default:
+			System.out.println("Please enter valid details.");
+		}
+	}
+
+	void totalDebitCreditOperaions(String bankName) {
+		switch (bankName) {
+		case "ICICI":
```

Comment:
here expetation was you will print like below, not in switch case. 
HDFC bank Credit Operation - 2
SBI bank Credit Operation - 1
BOB bank Credit Operation - 4